e2e/rdctl.e2e.spec.ts

```diff
@@ -834,6 +835,23 @@ test.describe('Command server', () => {
         }
       });
     });
+    test('should list images based on current containerEngine', async() => {
+      const { stdout, stderr, error } = await rdctl(['list-settings']);
+      const settings = JSON.parse(stdout);
```

Comment:
```suggestion
      const settings: Settings = JSON.parse(stdout);
```
It would be good to let TypeScript know what shape of object we expect.

Reply:
We aren't anywhere else in this file, but sure, why not.